h2o-core/src/main/java/water/Value.java

```diff
@@ -58,7 +58,7 @@
   // In any case, they will cause issues with both GC (giant pause times on
   // many collectors) and I/O (long term blocking of TCP I/O channels to
   // service a single request, causing starvation of other requests).
-  public static final int MAX = 256*1024*1024;
+  public static final int MAX = 1024*1024*1024; //DeepWater models can contain a single byte[] state as large as 1GB
```

Comment:
Minor: The best way would have a configuration option (secret one like `sys.ai.h2o.dkv.max.size`)